services/core-web/src/tests/components/common/comments/Comment.spec.tsx

```diff
@@ -1,9 +1,9 @@
 import React from "react";
-import { shallow } from "enzyme";
 import { Comment } from "@/components/common/comments/Comment";
+import { render } from "@testing-library/react";
 
-const dispatchProps = {};
-const props = {};
+const dispatchProps: any = {};
+const props: any = {};
```

Comment:
I think the props and dispatchProps here can just be omitted completely.

Reply:
The props are used, but I'll remove the dispatch